src/CLI/commands/build.ts

```diff
@@ -88,7 +88,7 @@ export = ts.identity<yargs.CommandModule<{}, Partial<ProjectOptions> & ProjectFl
 				default: false,
 				hidden: true,
 			})
-			// DO NOT PROVIDE DEFAULTS BELOW HERE, USE DEFAULT_PROJECT_OPTIONS
+			// Do not provide defaults below here, use DEFAULT_PROJECT_OPTIONS
```

Comment:
What's the purpose of this change?

Reply:
Removing unnecessary scream case in a comment